backend/config/environments/development/server.json

```diff
@@ -1,5 +1,5 @@
 {
-  "host": "localhost",
+  "host": "${process.env.APP_HOST || '0.0.0.0'}",
```

Comment:
Making 0.0.0.0 default instead of according to [Strapi migration recommendation](https://strapi.io/documentation/3.0.0-beta.x/migration-guide/migration-guide-beta.19-to-beta.19.4.html#listened-host-changed)